docs/core/tutorials/metapackages.md

```diff
@@ -0,0 +1,14 @@
+
```

Comment:
this is missing metadata

Reply:
This file was supposed to be removed (The information from it is now in the versions/index.md file.)